src/components/home/components/forms/AddMappingMetaForm.tsx

```diff
@@ -13,17 +16,17 @@ interface Props {
 };
 
 
-const AddMappingMetaForm = (props: Props) => {
+const AddMappingMetaForm = (props: Props, secondary?: boolean) => {
```

Comment:
I don't see the `secondary` being used, is that correct?

Reply:
No longer used, so removed